docs/spec.yaml

```diff
@@ -1096,3 +1490,48 @@ definitions:
         description: |
           For when action type is 'clone' - the name of the environment you wish
           to clone
+  Policy:
+    type: object
+    required:
+      - name
+      - definition
+    properties:
+      id:
+        type: integer
+        description: policy id
+        readOnly: true
+      name:
+        type: string
+        description: the name of the policy
+      description:
+        type: string
+        description: the spec to be processed
+      environments:
+        type: array
+        items:
+            $ref: '#/definitions/Environment'
```

Comment:
I think the environments array needs to be a type of string. Swagger is rendering it as a whole env model